x/stake/keeper/delegation.go

```diff
@@ -71,15 +69,41 @@ func (k Keeper) SetDelegation(ctx sdk.Context, delegation types.Delegation) {
 	store.Set(GetDelegationKey(delegation.DelegatorAddr, delegation.ValidatorAddr), b)
 }
 
-// remove the delegation
+// remove a delegation from store
 func (k Keeper) RemoveDelegation(ctx sdk.Context, delegation types.Delegation) {
 	store := ctx.KVStore(k.storeKey)
 	store.Delete(GetDelegationKey(delegation.DelegatorAddr, delegation.ValidatorAddr))
 }
 
 //_____________________________________________________________________________________
 
-// load a unbonding delegation
+// Return all unbonding delegations for a delegator. If maxRetrieve is supplied, the respective amount will be returned.
+func (k Keeper) GetUnbondingDelegations(ctx sdk.Context, delegator sdk.AccAddress,
+	maxRetrieve ...int16) (unbondingDelegations []types.UnbondingDelegation) {
+
+	retrieve := len(maxRetrieve) > 0
```

Comment:
we shouldn't allow for retrieve all to exist - meaning that this `retrieve` term should be taken out - see my comments on the old PR

Reply:
ok, I'll to get all the validators I'll have to use `params.MaxValidators` then